pkg/aws/cloud_config.go

```diff
@@ -12,9 +12,11 @@ const (
 	flagAWSAPIEndpoints  = "aws-api-endpoints"
 	flagAWSAPIThrottle   = "aws-api-throttle"
 	flagAWSVpcID         = "aws-vpc-id"
+	flagAWSVpcTagKey     = "aws-vpc-tag-key"
```

Comment:
shall we consider to have `aws-vpc-infer-from-tags` and make it a string map? 
seems that's would be more general as in my sample vpcs, i only have a `name`: `vpc-name` tag.

Reply:
@M00nF1sh thank you for your comments, it looks good to me. Please note that I am from team of Cathal and since he is currently away, I am taking over this request and changes based on your comments are now added into a new PR: https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/3656/files. please take a look into the same.